pupil_src/shared_modules/calibration_routines/optimization_calibration/setup.py

```diff
@@ -43,29 +43,53 @@
 shared_cpp_include_path = '../../../shared_cpp/include'
 singleeyefitter_include_path = '../../../capture/pupil_detectors/singleeyefitter'
 
-opencv_libraries = ['opencv_core']
-opencv_library_dir = '/usr/local/opt/opencv3/lib'
-opencv_include_dir = '/usr/local/opt/opencv3/include'
 
-if(not os.path.isfile(opencv_library_dir+'/libopencv_core.so')):
-    ros_dists = ['kinetic', 'jade', 'indigo']
-    for ros_dist in ros_dists:
-        ros_candidate_path = '/opt/ros/'+ros_dist+'/lib'
-        if(os.path.isfile(ros_candidate_path+'/libopencv_core3.so')):
-            opencv_library_dir = ros_candidate_path
-            opencv_include_dir = '/opt/ros/'+ros_dist+'/include/opencv-3.1.0-dev'
-            opencv_libraries = [lib + '3' for lib in opencv_libraries]
-            break
+
+if platform.system() == 'Windows':
+    libs = []
+    library_dirs = []
+    lib_spec = [[np.get_include(), ''],
+               ['C:\\work\\opencv3.1.0\\include', 'C:\\work\\opencv\\build\\x64\\vc14\\lib\\opencv_world310.lib'],
```

Comment:
@pointcontrols - paths are not correct (at least for the bundling VM). 

Please update to:
`C:\\work\\opencv3.1.0\\build\\include`
`C:\\work\\opencv3.1.0\\build\\x64\\vc14\\lib\\opencv_world310.lib`